staging/src/k8s.io/apimachinery/pkg/api/errors/errors.go

```diff
@@ -855,3 +855,38 @@ func (r *ErrorReporter) AsObject(err error) runtime.Object {
 	})
 	return &status.ErrStatus
 }
+
```

Comment:
I think this code can move back to client-go once streamtranslator does

Reply:
Done